docs/architecture.md

```diff
@@ -11,6 +11,52 @@ In pouch's roadmap, we set ecosystem embracing as a big target. To upper orchest
 
 ![Ecosystem Architecture](static_files/pouch_ecosystem_architecture.png)
 
+The ecosystem architecture may be a little bit complicated at first glance. Take it easy. We can get a thorough understanding of it from the following three dimensions.
+
+### Runtime Layer
+
+Runtime layer is located on the right-top of the architure picture. This dimension mainly focus on OCI-compatible runtimes supported in Pouch. These runtimes unifies specifications for standard on operating system process and application containers. Currently, Pouch will support four kinds of OCI-compatible runtimes:
```

Comment:
of the architure picture -> in ?
unifies -> unify
standard -> standards
Pouch will support -> supports